dataplane/apigen/docparser/docparser.go

```diff
@@ -112,25 +113,44 @@ func ParseSAIXMLDir() (*SAIInfo, error) {
 	if err != nil {
 		return nil, err
 	}
+	members := []MemberDef{}
 	for _, file := range files {
 		if !strings.HasSuffix(file.Name(), "xml") {
 			continue
 		}
-		if err := parseXMLFile(filepath.Join(xmlPath, file.Name()), i); err != nil {
+		if strings.Contains(file.Name(), "saimetadata") {
```

Comment:
Nit: probably make a func that checks the filename to skip files?

Reply:
will do if this needs another case